project/Build.scala

```diff
@@ -677,6 +677,7 @@ object Build {
     val dottyCompiler = jars("dotty-compiler")
     val args0: List[String] = spaceDelimited("<arg>").parsed.toList
     val decompile = args0.contains("-decompile")
+    val consumeTasty = args0.contains("-gen-semantic-db")
```

Comment:
nit: I'd write `-gen-semanticdb` and in in camel case it would be `GenSemanticdb` (or `SemanticdbGen`, ..)

Reply:
Actually I should remove this flag.